font-src/glyphs/letter/latin/lower-h.ptl

```diff
@@ -96,18 +97,37 @@ glyph-block Letter-Latin-Lower-H : begin
 	select-variant 'h' 'h'
 	link-reduced-variant 'h/descBase' 'h'
 	link-reduced-variant 'h/sansSerif' 'h' MathSansSerif
-	turned 'turnh' 0x265 'h' Middle (XH / 2) [MarkSet.p]
+
+	derive-glyphs 'hCircumflex' 0x125 'h' : lambda [src gr] : glyph-proc
+		local shift : Width + SB - Middle + HalfStroke * HVContrast
+		include [refer-glyph 'circumflexAbove']
+		include : ApparentTranslate shift ((-XH) + Ascender)
+		include [refer-glyph src]
+		include : MarkSet.b
+		set-base-anchor 'aboveBraceL' ((-Middle) + shift - markExtend / 2) aboveMarkMid
```

Comment:
Need to update the placement of `above` anchor too, since we may stack more marks on it.